cmd/derper/derper.go

```diff
@@ -87,6 +88,11 @@ var (
 var (
 	tlsRequestVersion = &metrics.LabelMap{Label: "version"}
 	tlsActiveVersion  = &metrics.LabelMap{Label: "version"}
+
+	// 64 hexadecimal lowercase digits.
```

Comment:
As currently written it's "at least" 64, do we want exactly?

Reply:
Probably a good idea, more bits is better entropy.